packages/playground/src/components/no-list-data.tsx

```diff
@@ -37,7 +37,7 @@ const NoListData: React.FC<NoListDataProps> = ({
       className={cn('w-full h-full flex flex-col place-content-center place-items-center', {
         'py-20 pb-24': insideTabView
       })}>
-      {iconName && <Icon name={iconName} size={iconSize} />}
+      {iconName && <Icon name={iconName as IconProps['name']} size={iconSize} />}
```

Comment:
instead of casting, set the type on line 8 as `Partial<IconProps['name'], '...'>`

Reply:
Fixed in https://github.com/harness/canary/pull/137